opencvpos.py

```diff
@@ -0,0 +1,377 @@
+__author__ = 'will_boiolinha'
+
+
+import numpy as np
+import cv2
+
+class OpencvPos():
+    """
+    Black magic of infinity opencv skils and food with lactose
+    """
+
+    def __init__(self):
+        self.tags_name = ['Y.png', 'L.png', 'AR.png']
+        # tags vector
+        self.tags = [cv2.imread(name) for name in self.tags_name]
+        # matrix vector
+        self.matrix_size = 40
+        self.mats = [self.create_mat_from_tag(tag, self.matrix_size) for tag in self.tags]
+
+        self.errors = [[100,0] for i in range(len(self.tags))]
+        self.goods = [0 for i in range(len(self.tags) + 1)]
```

Comment:
Can't you change it too this?
`self.errors = [[100,0] for i in self.tags]`
`self.goods = [0 for i in self.tags]` 
why are the sizes different?